Packs/AbnormalSecurity/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,18 @@
+
+#### Integrations
+##### Abnormal Security
+  - Features Added
```

Comment:
As discussed, Let's remove the commands examples and leave only the commands.
For example:
- Abuse Mailbox Information commands:
  - abnormal-security-list-abuse-mailbox-campaigns
  - abnormal-security-get-abuse-mailbox-campaign